src/driver/mod.rs

```diff
@@ -128,20 +125,16 @@ impl<S: sequencing::SequencingSource<EngineApi>> Driver<EngineApi, S> {
             unsafe_block_signer_sender,
             network_service: Some(service),
             channel_timeout: config.chain.channel_timeout,
-            sequencing_src,
         })
     }
 }
 
-impl<E: Engine, S: sequencing::SequencingSource<E>> Driver<E, S> {
+impl<E: Engine> Driver<E> {
     /// Runs the Driver
     pub async fn start(&mut self) -> Result<()> {
-        tracing::trace!("starting driver, waiting for engine...");
-        self.await_engine_ready().await;
-        tracing::trace!("engine ready; starting chain watcher...");
+        tracing::info!("starting chain watcher...");
```

Comment:
change back to trace. i believe you can set env var `RUST_LOG=tracing` to print tracing logs

Reply:
yeah it's just for convenience but i can change it back